### ISSUE TITLE
Always get a SI_Error = -3 when trying to read a simple ini file

### ISSUE THREAD

**Issue by BanditBloodwyn** [brofield/simpleini#34]
Hey guys,

I have a (surely simple to solve) problem with reading a simple ini-file.
Our parser has no problem to find and open the file, but always gives me the SI_FILE error.
How can I fix this? What information do you need to help me?

**Comment by brofield** [comment]
Any time you get an error you don't understand, check the source code and/or step through it. In this case I would check errno...

https://github.com/brofield/simpleini/blob/master/SimpleIni.h#L245
    SI_FILE     = -3    //!< File error (see errno for detail error)

**Comment by BanditBloodwyn** [comment]
Hey,
sorry, I forgot to give you a better desciption.
I went through the LoadFile function during runtime and saw, that the problem occurs in line 1399 ( if (uRead != (size_t) lSize) ). uRead and lSize are unequal for my ini-file and I don't know why.

If the problem lies in the ini-file itself, you can find it here:
https://github.com/BanditBloodwyn/PatricianX/blob/master/System/ini/config.ini

**Comment by brofield** [comment]
Are running this on Windows, calling LoadFile with a FILE* and have opened the file in text mode rather than binary mode? CRLF conversion could account for it.

**Comment by BanditBloodwyn** [comment]
Yes, I'm running on Windows 10. I wrote that file with the Notepad++. How that should be a problem?

**Comment by BanditBloodwyn** [comment]
I found the problem. It's a bit strange, because in my previous tests the parser FOUND the file but didn't process it correctly (the "size-problem" I meant). Now I tried to start my program NOT from the studio but by the exe itself. It worked. 
So I changed the working directory of my studio to the Debug directory (where the exe and the ini files are in). Now everything works fine (finding, loading and reading). 

Thanks for your help!
